src/transformers/modeling_albert.py

```diff
@@ -651,7 +651,7 @@ def forward(
         head_mask=None,
         inputs_embeds=None,
         labels=None,
-        sentence_order_label=None,
+        sentence_order_labels=None,
```

Comment:
While I understand that `sentence_order_labels` might be more appropriate, renaming this would be a breaking change. Do you mind switching back to the previous naming scheme?

Reply:
switched to the previous naming.